### PR TITLE
add BadgeSection

### PR DESCRIPTION
**Pull request by MixailE**

### CODE REVIEW

File: src/components/ContractName.tsx [hunk 1]
```diff
@@ -2,19 +2,35 @@ import { Suspense, memo } from 'react'
...
+  truncate,
+  clearType,
+}: ContractNameProps) {
+  const { emailDerivativeContracts = [], ERC721derivativeContracts = [] } =
+    useSnapshot(SealCredStore)
   const { contractNames } = useSnapshot(ContractNamesStore)
-  const contractName = contractNames[address]
-  if (!contractNames[address]) ContractNamesStore.fetchContractName(address)
+  let contractName = contractNames[address]
+  if (!contractName) ContractNamesStore.fetchContractName(address)
```
Comment (T-Damer) [inline src/components/ContractName.tsx:25]: What if there's no `contractName`, then it tries to fetch it, no `contractName` again, you'll get an infinite loop here?

File: src/components/badges/BadgeSection.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import ChildrenProp from 'models/ChildrenProp'
+import classnames, {
+  display,
+  gap,
+  gridAutoRows,
+  gridTemplateColumns,
+} from 'classnames/tailwind'
+
+const badgesList = classnames(
+  display('grid'),
```
Comment (T-Damer) [inline src/components/badges/BadgeSection.tsx:12]: We use same styles for in `List.tsx`, maybe it's better to reuse them?

**Review by T-Damer — CHANGES_REQUESTED** [review]
Mostly fine, let's check if we can make it better :)

File: src/components/ContractName.tsx (hunk 1 above)
Comment (MixailE) [reply src/components/ContractName.tsx:25]: No, the `contractName` will be requested only once)

File: src/components/badges/BadgeSection.tsx (hunk 2 above)
Comment (MixailE) [reply src/components/badges/BadgeSection.tsx:12]: They are no longer in `List.tsx` 🤔

**Review by T-Damer — APPROVED** [review]
lgtm

**Review by smyzin — APPROVED** [review]
Looks good

**Pull request merged**